member.php

```diff
@@ -1208,63 +1265,66 @@
 			// JS validator extra
 			$lang->js_validator_password_length = $lang->sprintf($lang->js_validator_password_length, $mybb->settings['minpasswordlength']);
 
+			$validator_javascript .= "
+	$.validator.addMethod('passwordSecurity', function(value, element, param) {
+		return (value != $('#email').val() && value != $('#username').val()
+			&& value.indexOf($('#email').val()) === false && value.indexOf($('#username').val()) === false
+			&& $('#email').val().indexOf(value) === false && $('#username').val().indexOf(value) === false);
+	}, '{$lang->userdata_bad_password_security}');\n";
```

Comment:
This language string is not available in this position. Preparing patch...

Reply:
Oops, good catch!

Reply:
Fixed in #2878.